packages/ndla-ui/src/LicenseByline/EmbedByline.tsx

```diff
@@ -28,9 +27,6 @@ interface BaseProps {
   children?: ReactNode;
   visibleAlt?: string;
   error?: true | false;
-  hideOnLargeScreens?: boolean;
-  first?: boolean;
-  inGrid?: boolean;
```

Comment:
Har du testa dette? inGrid er lagt inn av en grunn meiner eg å hugse.

Reply:
Yup, den er brukt for skallering ned på mindre skjermstørrelser. Men ting blir annerledes nå så derfor ble den slettet